src/rpc/mempool.cpp

```diff
@@ -644,6 +672,30 @@ static RPCHelpMan gettxspendingprevout()
                 const CTransaction* spendingTx = mempool.GetConflictTx(prevout);
                 if (spendingTx != nullptr) {
                     o.pushKV("spendingtxid", spendingTx->GetHash().ToString());
+                } else if (mempool_only.value_or(false)) {
+                    // do nothing
+                } else if (g_txospenderindex) {
+                    // no spending tx in mempool, query txospender index
+                    uint256 spendingtxid;
+                    if(g_txospenderindex->FindSpender(prevout, spendingtxid)) {
```

Comment:
nit (only if you retouch):
```suggestion
                    if (g_txospenderindex->FindSpender(prevout, spendingtxid)) {
```

Reply:
done in https://github.com/bitcoin/bitcoin/pull/24539/commits/965bc7ef2e8548fd9abe609a752cadf7c44b8c00